server/game/cards/23-AHaH/LordProtectorOfTheVale.js

```diff
@@ -0,0 +1,41 @@
+const DrawCard = require('../../drawcard.js');
+
+class DefensiveDebris extends DrawCard {
```

Comment:
Name needs to be changed (to LordProtectorOfTheVale). Same for the imports at the bottom.

Reply:
Fixed myself in [3e1ed6b](https://github.com/throneteki-playtesting/throneteki/pull/76/commits/3e1ed6b8f90d60b5e26adf79694f7228c5d076fc).